ingest/change.go

```diff
@@ -327,3 +331,658 @@ func (c Change) AccountChangedExceptSigners() (bool, error) {
 
 	return !bytes.Equal(preBinary, postBinary), nil
 }
+
+func (c Change) ExtractEntry() (xdr.LedgerEntry, xdr.LedgerEntryChangeType, bool, error) {
+	switch changeType := c.LedgerEntryChangeType(); changeType {
+	case xdr.LedgerEntryChangeTypeLedgerEntryCreated, xdr.LedgerEntryChangeTypeLedgerEntryUpdated:
+		return *c.Post, changeType, false, nil
+	case xdr.LedgerEntryChangeTypeLedgerEntryRemoved:
+		return *c.Pre, changeType, true, nil
+	default:
+		return xdr.LedgerEntry{}, changeType, false, fmt.Errorf("unable to extract ledger entry type from change")
+	}
+}
+
+func (c Change) Deleted() (bool, error) {
+	_, _, deleted, err := c.ExtractEntry()
+	if err != nil {
+		return false, err
+	}
+
+	return deleted, nil
```

Comment:
seems like this could just be
```suggestion
	return c.LedgerEntryChangeType() == xdr.LedgerEntryChangeTypeEntryRemoved
```
and not have an error case

Reply:
Ah that's fair. Yeah instead of the whole using `ExtractEntry()` I could just get the data directly